addon_service/configured_addon/computing/views.py

```diff
@@ -0,0 +1,9 @@
+from addon_service.configured_addon.views import ConfiguredAddonViewSet
+
+from .models import ConfiguredComputingAddon
+from .serializers import ConfiguredComputingAddonSerializer
+
+
+class ConfiguredComputingAddonViewSet(ConfiguredAddonViewSet):
+    queryset = ConfiguredComputingAddon.objects.active()
+    serializer_class = ConfiguredComputingAddonSerializer
```

Comment:
If you are planning to fetch credentials from `gravyvalet` for computing addons (in this PR), you should add action for credential fetching similar to `ConfiguredStorageAddonViewSet`